sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -206,7 +718,7 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
     // Set the CAS supplier.
     await db.student.update(
       { id: student.id },
-      { casSupplier, updatedAt: expectedUpdatedDate },
+      { casSupplier, updatedAt: expectedUpdatedDate ?? new Date() },
```

Comment:
the value for new date is mocked. so if the intention is to set student updated out outside the delta range, pass a date accordingly to the method createStudentWithExpectedData()